index.html

```diff
@@ -8,46 +8,55 @@
   <meta name="author" content="Wolox">
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
-<body class="list-container">
-  <div class="list-item">
-    <a class="item-book-cover" href="detail.html"></a>
-    <span class="item-bold-text">Título del libro</span>
-    <span class="item-text">Autor del libro</span>
-  </div>
-  <div class="list-item">
-    <a class="item-book-cover" href="detail.html"></a>
-    <span class="item-bold-text">Título del libro</span>
-    <span class="item-text">Autor del libro</span>
-  </div>
-  <div class="list-item">
-    <a class="item-book-cover" href="detail.html"></a>
-    <span class="item-bold-text">Título del libro</span>
-    <span class="item-text">Autor del libro</span>
-  </div>
-  <div class="list-item">
-    <a class="item-book-cover" href="detail.html"></a>
-    <span class="item-bold-text">Título del libro</span>
-    <span class="item-text">Autor del libro</span>
-  </div>
-  <div class="list-item">
-    <a class="item-book-cover" href="detail.html"></a>
-    <span class="item-bold-text">Título del libro</span>
-    <span class="item-text">Autor del libro</span>
-  </div>
-  <div class="list-item">
-    <a class="item-book-cover" href="detail.html"></a>
-    <span class="item-bold-text">Título del libro</span>
-    <span class="item-text">Autor del libro</span>
-  </div>
-  <div class="list-item">
-    <a class="item-book-cover" href="detail.html"></a>
-    <span class="item-bold-text">Título del libro</span>
-    <span class="item-text">Autor del libro</span>
-  </div>
-  <div class="list-item">
-    <a class="item-book-cover" href="detail.html"></a>
-    <span class="item-bold-text">Título del libro</span>
-    <span class="item-text">Autor del libro</span>
+<body>
+  <nav class="navbar">
+    <ul class="navbar-nav">
+      <li><a href="#" class="logo"></a></li>
+      <li><a href="#"><span>Logout</span></a></li>
+    </ul>
+  </nav>
```

Comment:
```suggestion
  <nav class="navbar">
      <a href="#" class="logo"></a>
      <a href="#"><span>Logout</span></a>
  </nav>
```
i think that it looks better thus. Okey, you should think now: "but... and the side spaces?" haha. Use the same styles that you used for the [container list](https://github.com/wolox-training/st-angular/blob/d36cdc7d09cd35f684a9d5c9316eb8c5f47e9122/scss/index.scss#L5)

Reply:
I speak about `padding`

Reply:
I can use `navbar ` and `navbar-nav` in a same tag and work fine